content/en/synthetics/settings/_index.md

```diff
@@ -35,13 +35,77 @@ further_reading:
 
 On the [Synthetic Monitoring & Continuous Testing Settings page][1], you can access and control the following topics:
 
+* [Default Settings](#default-settings)
 * [Private Locations](#private-locations)
 * [Global Variables](#global-variables)
-* [Default Settings](#default-settings)
 * [Integration Settings](#integration-settings)
 * [Continuous Testing Settings][2]
 * [Mobile Applications Settings][18]
 
+## Default settings
+
+### Enforced tags settings
+
+Selecting this option ensures that configured tags from the Usage Attribution tab are required by users creating and editing Synthetic tests. They won't be able to save tests without entering all required tags.
+
+Usage Attribution tags allow you to break down cost and usage attributes of this feature by services, applications, or teams. 
+
+### Default locations
+
+Choose the default locations for your [API test][4], [multistep API test][5], or [browser test][6] details. 
+
+Your options include all of the available managed locations Datadog offers and the private locations you set up for your account.
+
+When you are done selecting locations, click **Save Default Locations**.
+
+### Default browsers and devices
+
+Choose the default browser and device types for your [browser test][6] details.
+
+Your options for browsers include Google Chrome, Firefox, and Microsoft Edge. Your options for devices include a large laptop, a tablet, and a small mobile device.
+
+When you are done selecting browsers and devices, click **Save Default Browsers & Devices**.
+
+### Default tags
+
+Choose or add the default tags for your [API test][4], [multistep API test][5], or [browser test][6] details.
+
+When you are done selecting related tags, click **Save Default Tags**.
+
+### Permissions
+
+By default, only users with the [Datadog Admin and Datadog Standard roles][11] can access the Synthetic Monitoring **Default Settings** page. To get access to the **Default Settings** page, upgrade your user to one of those two [default roles][11]. 
+
+If you are using the [custom role feature][12], add your user to any custom role that includes `synthetics_default_settings_read` and `synthetics_default_settings_write` permissions. 
+
+## Integration settings
+
+{{< img src="synthetics/settings/integration_settings.png" alt="Integration Settings page" style="width:100%;">}}
+
+### APM integration for browser tests
+
+Allow URLs to add APM integration headers to those URLs. Datadog's APM integration headers allow Datadog to link browser tests with APM. 
```

Comment:
```suggestion
Datadog's APM integration headers allow Datadog to link browser tests with APM. 
```

removing first sentence since it's restated with more clarity in the next line